experiments/where-do-users-find-our-sites.md

```diff
@@ -0,0 +1,103 @@
+# Where do users find out about Alaveteli sites?
+
+Where is our existing traffic being referred from? Where might there be
+potential users that we're not currently reaching?
+
+- [issue#4](https://github.com/mysociety/alaveteli-experiments/issues/4)
+
+## Answer
+
+**TL;DR:** so far, referrer analysis reveals a dominance of social media sites
+for most of our sites with more of a news/campaign-driven audience for the more
+journalistic sites such as Ask The EU. Inbound link analysis inconclusive due to
+the limited features of free reports.
+
+## Analysis
+
+This has turned out to be a more complex question than we first realised so we're
+going to try to update this as we find out more.
+
+We have two types of data available:
+
+### [Referrer data](where-do-users-find-our-sites/inbound-traffic.md)
+
+Data extracted from Google Analytics showing which links the people who are
+visiting our sites have clicked to reach us.
+
+From studying this data we appear to have two types of site - those which get
```

Comment:
>
```ruby
# => { "Social Media"   => { :average => 35.54, :max => 56.13, :min => 10.0 },
#      "Civic Tech"     => { :average =>  7.05, :max => 18.68, :min =>  6.78 },
#      "Portal"         => { :average =>  3.78, :max => 14.47, :min =>  4.45 },
#      "Campaign Group" => { :average =>  2.4,  :max => 12.01, :min => 12.01 },
#      "Forum"          => { :average =>  1.59, :max =>  4.46, :min =>  3.51 },
#      "News"           => { :average =>  1.15, :max =>  5.77, :min =>  5.77 },
#      "Search Engine"  => { :average =>  0.54, :max =>  2.69, :min =>  2.69 },
#      "Knowledge Base" => { :average =>  0.43, :max =>  2.16, :min =>  2.16 } }
```

Same process run with "current" data - very little change at the top but News appears to be rallying:

```ruby
# => { "Social Media"    => { :average => 33.96, :max => 50.15, :min => 14.49 }, 
#      "Civic Tech"      => { :average =>  9.39, :max => 31.09, :min =>  4.82 },
#      "News"            => { :average =>  7.43, :max => 20.78, :min => 16.35 },
#      "Campaign Group"  => { :average =>  4.61, :max => 23.06, :min => 23.06 },
#      "Portal"          => { :average =>  2.16, :max => 10.81, :min => 10.81 },
#      "App"             => { :average =>  2.12, :max => 10.6,  :min => 10.6  },
#      "Search Engine"   => { :average =>  1.32, :max => 6.58,  :min =>  6.58 },
#      "Gov Data Portal" => { :average =>  1.2,  :max => 6.01,  :min =>  6.01 },
#      "Forum"           => { :average =>  0.57, :max => 2.86,  :min => 2.86  } }
```